packages/core/server/util/enabledContentTypes.ts

```diff
@@ -0,0 +1,16 @@
+import _ from 'lodash';
+import { pluginId } from './pluginId';
+
+export const isContentTypeEnabled = (uid: string) => {
+  const contentType = strapi.contentTypes[uid];
+  const { pluginOptions } = contentType;
+
+  const isInContentManager = _.get(pluginOptions, ['content-manager', 'visible']);
+  if (isInContentManager === false) return false;
```

Comment:
Is this check actually needed now that url alias isn't enabled by default?

```suggestion
```

Reply:
Yep. You're right.
Done.